vpn_slice/linux.py

```diff
@@ -93,6 +94,11 @@ def deconfigure_firewall(self, device):
 
 
 class CheckTunDevProvider(TunnelPrepProvider):
+    def create_tunnel(self):
+        node = '/dev/net/tun'
+        if not os.path.exists(path):
+            os.makedirs(os.path.dirname(node), exist_ok=True)
+            os.mknod(node, mode=0o640 | os.stat.S_IFCHR, device = os.makedev(10, 200))
```

Comment:
This doesn't make sense. `path` isn't defined, and `os.stat` is a function, not at all the same thing as the `stat` module.

Looks like you did a bad merge in 73366f7dc87622df0c0a1466baa8bbcc54e407ff. Please fix and squash out the bad commits.

Reply:
Yep. My git-kungfu is not very good. And `os.stat` doesn't have S_IFCHR.

Reply:
No worries. The only mistake you made was merging in *my* code. :joy: